### ISSUE TITLE
It would be nice if there was the ability to move panes on the sia hostd web windows

### ISSUE THREAD

**Issue by rdwild** [SiaFoundation/hostd#62]
### Description

It would be nice if there was the ability to move panes on the sia hostd web windows.

Right now at the to is revenue so it would be nice to move the contract pane to the top rather than have revenue at the top.

<img width="1385" alt="image" src="https://github.com/SiaFoundation/hostd/assets/44580802/03807af0-9168-4698-a7a2-26c2c2360e0a">

it would also be nice if we could ad some widgets similar to the ones that show up at the bottom of the sia host manager apps.

### Version

0.1.0 Alpha 1

### What operating system are you running (e.g. Ubuntu 22.04, macOS, Windows 11)?

Ubuntu 22.04, windows 10 chrome.

**Comment by alexfreska** [comment]
@rdwild I agree that the visibility and arrangement of the graphs and stat cards should all be configurable 👍🏼 Will check back in here when I get around to a prototype.
